app/controllers/books_controller.rb

```diff
@@ -7,13 +7,38 @@ def new
     @book = Book.new
   end
 
+  def edit
+    @book = Book.find(params[:id])
+  end
+
   def create
-    @book = Book.create(book_params)
-    redirect_to books_path
+    @book = Book.new(book_params)
+    if @book.save
+      flash[:success] = t('books.create.flash_success')
+      redirect_to books_path
+    else
+      render :new
```

Comment:
I would add a test for this as well so you know that it works. Once that's done I think it'll be good to go!